packages/amplify-util-mock/src/__tests__/mockAll.test.ts

```diff
@@ -0,0 +1,45 @@
+import { prompter } from '@aws-amplify/amplify-prompts';
+import { mockAllCategories } from '../mockAll';
+import { start as apiServerStart } from '../api';
+import { start as storageServerStart } from '../storage';
+import { start as lambdaServerStart } from '../func';
+import { $TSAny } from 'amplify-cli-core';
```

Comment:
This should now be `@aws-amplify/amplify-cli-core`. Please make sure you're working off the latest changes from dev.

Reply:
modified it to `@aws-amplify/amplify-cli-core` and yarn tested for any failures